### DISCUSSION TITLE
Display images in list view

### DISCUSSION THREAD

**Discussion by upieez** [dailydotdev/daily#912] [Feature Requests]
Currently, only card view shows images and it would be nice to have a similar image displayed for list view too! It would allow the content to be more engaging, similar to how reddit does it. 

Would be happy to give it a shot and contribute via code if possible too :) 

<img width="1359" alt="Screenshot 2023-06-08 at 5 29 22 PM" src="https://github.com/dailydotdev/daily/assets/56812343/cb323dbd-c548-46d7-afd7-db7be932b445">

**Comment by idoshamun** [comment]
Hi @upieez,

Thanks for the suggestion. We're now in the proceed of rethinking our feed layout a bit.
May take me some time to reply on this one.

**Reply by upieez** [reply]
no worries! thanks for updating me about the feed layout too. Excited to see what the next phase of daily.dev is going to be
